io.github.Abstract_AA.Framestop.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.Abstract_AA.Framestop</id>
+  
+  <name>Framestop</name>
+  <summary>Freeze frame tool that auto selects the best frame</summary>
+
+  <icon type="remote">https://raw.githubusercontent.com/Abstract-AA/Framestop/refs/heads/main/src/io.github.Abstract_AA.Framestop.svg</icon>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+
+  <developer id="io.github.abstract_aa">
+    <name>Arthur Afonso and Felipe Simões</name>
+    <name xml:lang="pt">Arthur Afonso e Felipe Simões</name>
+  </developer>
+
+  <url type="homepage">https://github.com/Abstract-AA/Framestop/</url>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </supports>
+
+  <content_rating type="oars-1.1" />
+  
+  <description>
+    <p>
+      Framestop is a Flatpak application designed to extract high-quality freeze frames from videos. Using advanced image analysis, the app identifies and selects the most representative and visually distinct frames from the video.
+    </p>
+    <p>
+      <em>How It Works</em>
+    </p>
+    <p>
+      1. User loads the video and selects a frame on the timeline.
+    </p>
+    <p>
+      2. Five neighboring frames from the current frame are selected for analysis (this number can be changed in configs).
+    </p>
+    <p>
+      3. The software calculates perceptual color differences between neighboring pixels using the CIELAB color space.
+    </p>
+    <p>
+      4. The frame with the most perceptual pixel transitions (more details) is selected for the screenshot.
+    </p>
```

Comment:
Use li and ul tags https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#description